packages/html/src/card/templates/card-normal.tsx

```diff
@@ -0,0 +1,7 @@
+import { Card } from '../../card';
+
+export const CardNormal = (props) => (
+    <Card orientation="vertical"
```

Comment:
We should add a `children` prop here with a default template example. 

Same goes for **templates/card-horizontal.tsx**.